### ISSUE TITLE
Review Windows Accessiblity Audits (JAWs, Zoomtext, Dragon: Footer, Key details, Toggle Welsh)

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#421]
## What
Let's review the audit result to understand what the problems and potential fixes are. We'll need to consider whether working in this way is even possible and explore the potential of getting the tools to test with on the team.

## Done when

- [ ] Review fails
- [ ] Contact accessibility team for clarification 
- [ ] Raise issues to fix
- [ ] Amend accessibility statement to cover any won't/can't fix issues

## Outcomes
WCAG compliance and no known barriers for teams using components.

### Publish accessibility summary for each thing:
- https://github.com/dwp/design-system/issues/295
- https://github.com/dwp/design-system/issues/365

**Comment by jonhurrell** [comment]
Emailed Nicki to start process of getting windows assistive devices and software.

**Comment by jonhurrell** [comment]
Sharon going to arrange a demo when JAWs issues fixed (29/09).
